src/Pact/Repl.hs

```diff
@@ -200,6 +200,11 @@ handleCompile :: String -> Exp Parsed -> (Term Name -> Repl (Either String a)) -
 handleCompile src exp a =
     case compile (mkStringInfo src) exp of
       Right t -> a t
+      -- special case for `with-capability` bareword due to
+      -- `with-capability` being a reserved word that fails to
+      -- compile if issued in the repl for doc purposes
+      Left{} | "with-capability" == (exp ^. _EAtom . to _atomAtom) ->
```

Comment:
I wonder if there's any behavior here that doesn't work the way we want given that `_EAtom` with this accessor works as a fold.

Also btw we can make this work on any other native that has a special form. I'll follow up about this in a bit.

Reply:
> I wonder if there's any behavior here that doesn't work the way we want given that _EAtom with this accessor works as a fold.

that was a question for me too. I suspect it may be tighter if we use `info` as well. 

> Also btw we can make this work on any other native that has a special form. I'll follow up about this in a bit.

nice that'd be good.